### DISCUSSION TITLE
Exclude Containers for Notification?

### DISCUSSION THREAD

**Discussion by DocMAX** [containrrr/watchtower#1644] [General]
Can i exclude containers for notifications? Just notifications, not the update.

**Comment by derN3rd** [comment]
Yeah, would be great if we can add a second label to the contains we want to exclude notifications from.

I have a few containers that get nightly builds, but I don't want to get information about them being updated every day

**Reply by derN3rd** [reply]
@DocMAX, I found a workaround until this is implemented:
https://containrrr.dev/watchtower/running-multiple-instances/

You can run multiple instances with different scopes. By this you can just enable the notifications for scope1 and put all containers where you don't want notifications from to the second scope
